app/src/main/java/com/wut/indoornavigation/logic/graph/impl/HeuristicFunction.java

```diff
@@ -0,0 +1,16 @@
+package com.wut.indoornavigation.logic.graph.impl;
+
+import com.wut.indoornavigation.logic.graph.models.Vertex;
+
+import static java.lang.Math.sqrt;
+import static java.lang.Math.abs;
+import static java.lang.Math.pow;
+
+public final class HeuristicFunction {
```

Comment:
@Singleton + dodaj konstruktor z adnotacja @Inject, a potem wstrzyknij to do GraphImpl